main.js

```diff
@@ -1,144 +1,225 @@
-let darkMode = document.querySelector(".themes");
-let darkModeToggle = document.querySelector(".themes .fa-moon")
-let navBar = document.querySelector('.nav-bar')
-let textInDark = document.querySelectorAll('.text-dark')
-let main = document.getElementById("mainSec")
-let searchBar = document.getElementById('search')
-let listOfCountries = document.getElementById("options")
-let filter = document.getElementById('filterBar')
-let cards = document.getElementsByClassName("cards-styling")
-console.log(cards)
-let httpReq = new XMLHttpRequest()
-httpReq.open("GET", "data.json");
-httpReq.send()
+// let darkMode = document.querySelector(".themes");
+// let darkModeToggle = document.querySelector(".themes .fa-moon");
+// let navBar = document.querySelector('.nav-bar');
+// let textInDark = document.querySelectorAll('.text-dark');
+// let main = document.getElementById("mainSec");
+// let searchBar = document.getElementById('search');
+// let listOfCountries = document.getElementById("options");
+// let filter = document.getElementById('filterBar');
+// let cards = document.getElementsByClassName("cards-styling");
+let darkMode = document.querySelector(".themes"),
+    darkModeToggle = document.querySelector(".themes .fa-moon"),
+    navBar = document.querySelector(".nav-bar"),
+    textInDark = document.querySelectorAll(".text-dark"),
+    main = document.getElementById("mainSec"),
+    searchBar = document.getElementById("search"),
+    listOfCountries = document.getElementById("options"),
+    filter = document.getElementById("filterBar"),
+    cards = document.getElementsByClassName("cards-styling");
+
+// let httpReq = new XMLHttpRequest()

```

Comment:
20 : 32 => -1

This method of creating requests is old and not efficient. the right and easiest way is "fetch"